cmd/uraggregate/main.go

```diff
@@ -62,15 +62,15 @@ func runAggregation(db *sql.DB) {
 
 	since = maxIndexedDay(db, "Performance")
 	log.Println("Aggregating Performance data since", since)
-	rows, err = aggregatePerformance(db, since)
+	rows, err = aggregatePerformance(db, since.Add(24*time.Hour))
 	if err != nil {
 		log.Println("aggregate:", err)
 	}
 	log.Println("Inserted", rows, "rows")
 
 	since = maxIndexedDay(db, "BlockStats")
 	log.Println("Aggregating BlockStats data since", since)
-	rows, err = aggregateBlockStats(db, since)
+	rows, err = aggregateBlockStats(db, since.Add(24*time.Hour))
```

Comment:
I think the since here should be now().add(24 hours).truncate(24 hours) for the > comparisons in the sql to be correct

Reply:
Maybe that’s equivalent to what’s already done since “since” should be a midnight timestamp…

Reply:
That's the idea, that is why I've added the truncate on getting the since.